backend/Dockerfile

```diff
@@ -8,12 +8,11 @@ RUN apt-get update \
 
 RUN pip install pipenv
 
-# copy pipfile
-COPY [ "Pipfile" ,"Pipfile.lock","./"]
-
-# set up packages 
-RUN pipenv install --system --deploy
+# copy the requirements txt file
+COPY requirements.txt requirements.txt
 
+# install the dependencies
+RUN pip3 install -r requirements.txt
 
 # set working directory
 WORKDIR /app
```

Comment:
actually if you follow the dockerfile, the entrypoint.sh is at the root of the copyied files since i have changed to the working directory which is /app, am not sure if it will work with the absolute path